tests/components/kraken/test_config_flow.py

```diff
@@ -0,0 +1,121 @@
+"""Tests for the kraken config_flow."""
+from unittest.mock import patch
+
+from homeassistant.components.kraken import config_flow
+from homeassistant.components.kraken.const import DOMAIN
+from homeassistant.const import CONF_SCAN_INTERVAL
+
+from .const import TICKER_INFORMATION_RESPONSE, TRADEABLE_ASSET_PAIR_RESPONSE
+
+from tests.common import MockConfigEntry
+
+
+def init_config_flow(hass):
+    """Init a configuration flow."""
+    flow = config_flow.KrakenConfigFlow()
+    flow.hass = hass
+    return flow
+
+
+async def test_config_flow(hass):
+    """Test we can finish a config flow."""
+    with patch(
+        "pykrakenapi.KrakenAPI.get_tradable_asset_pairs",
+        return_value=TRADEABLE_ASSET_PAIR_RESPONSE,
+    ):
+        with patch(
```

Comment:
We don't need two context managers for the two patches since we want the same scope for them. Just comma separate the `patch` calls within the `with` statement.

Reply:
Implemented except for `test_missing_pair_marks_sensor_unavailable` where I need to change the patch later on